### ISSUE TITLE
"open file on remote" command is gone 

### ISSUE THREAD

**Issue by shogunsea** [gitkraken/vscode-gitlens#2748]
### Description

after upgrading the VS to `May 2023 (version 1.79)` , it seems also upgraded gitlens to `Version [14.0.0]`
but I found the `open file on remote` command is gone, seems like a bug? 
`open file from remote` is still there, but it's asking a full url, which isn't useful at all. 

### GitLens Version

14

### VS Code Version

Version: 1.79.2 (Universal)
Commit: 695af097c7bd098fbf017ce3ac85e09bbc5dda06
Date: 2023-06-14T08:58:52.392Z
Electron: 22.5.7
Chromium: 108.0.5359.215
Node.js: 16.17.1
V8: 10.8.168.25-electron.0
OS: Darwin arm64 22.5.0

### Git Version

git version 2.39.2

### Logs, Screenshots, Screen Captures, etc

<img width="635" alt="image" src="https://github.com/gitkraken/vscode-gitlens/assets/2238944/d3fa5311-3644-4190-adc8-a79ba81d7898">

**Comment by shogunsea** [comment]
hmm weirdly it's showing up now
<img width="633" alt="image" src="https://github.com/gitkraken/vscode-gitlens/assets/2238944/e9918001-3097-4fed-a858-8e2794436157">

I guess sometimes the plugin might be flaky..?

**Comment by eamodio** [comment]
You have to have an active file for the command to be available.

**Comment by shogunsea** [comment]
>  You have to have an active file for the command to be available.
yeah that's how I tested in both cases, the screenshot didn't capture the file.  
Feel free to close it as this seems to be some kind of flake.
